src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1003.UrisMustBeValid_Invalid.sarif

```diff
@@ -20,7 +20,18 @@
           "rules": [
             {
               "id": "TST0001",
-              "helpUri": "ht%tp://www.example.com/rules/tst0001.html"
+              "helpUri": "ht%tp://www.example.com/rules/tst0001.html",
+              "shortDescription": {
+                "text": "Some testing occurred."
+              },
+              "fullDescription": {
+                "text": "Some testing occurred. Full text."
+              },
+              "messageStrings": {
+                "RuleId": {
```

Comment:
>RuleId [](start = 17, length = 6)

Let's use `Default` for the generic message string id. `RuleId` comes from `SARIF1020`, and is not applicable here.

Reply:
ok. i'm updating everywhere to default instead of ruleid.